.github/workflows/deploy.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
-            ${{ runner.os }}-node-
+            ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
```

Comment:
I don't actually understand why this is needed, but I trust you 😂

Reply:
😆 hahaha thanks! It's just to make sure we restore cached dependencies correctly, i.e. by naming the cache artifacts after the hashed contents of package-lock.json, which will prevent saving/restoring an old/outdated cache. I believe caches are stored per branch, so this only affects individual branches, and only if you add/update/remove a dependency.